src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -79,28 +77,13 @@ public void c024928()
             var tree = SyntaxFactory.ParseSyntaxTree(test);
         }
 
+#if !DEBUG 
```

Comment:
I suggested to use conditional compilation because the custom fact would also be implemented using conditional compilation, but in a different project. It felt natural to me to simply \#if the unit-test rather than build whole infrastructure to accomplish the same thing.